docs/dapp/emerald/README.mdx

```diff
@@ -1,107 +1,94 @@
 import DocCardList from '@theme/DocCardList';
 import {findSidebarItem} from '@site/src/sidebarUtils';
+import {AddEmeraldToMetaMask as E, AddEmeraldTestnetToMetaMask as ET} from '@site/src/AddToMetaMask';
```

Comment:
If you're renaming components just to make it fit into markdown tables, you can also:

```mdx
import {AddEmeraldToMetaMask, AddEmeraldTestnetToMetaMask as ET} from '@site/src/AddToMetaMask';

export const AddEmerald1RPCtoMM = () => (<AddEmeraldToMetaMask rpcs={['https://1rpc.io/oasis/emerald']} />);

<!-- <E rpcs={['https://1rpc.io/oasis/emerald']} /> -->
<AddEmerald1RPCtoMM />
```

Reply:
Hm, I'm not in fond of this because then this would require editing in two places when you want to remove/add/rename a RPC. The reason why I added a new component originally was because I didn't want the "Add to MetaMask" code being duplicated in markdown for each RPC.